testdata/repos/standalone/PackageMetadataXmlCheck/InvalidRemoteID/metadata.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>maintainer@random.email</email>
+	</maintainer>
+	<upstream>
```

Comment:
All those remotes test very nicely the `_two_components_validator_re` regex. But we are missing for all other regexes. Let's add some failing examples (in this file) for all other regexes.

Reply:
I said this was WIP. I've finished adding all examples, now grouped into 1/2/2+ component cases.